openwisp_monitoring/device/base/models.py

```diff
@@ -384,12 +385,22 @@ def related_metrics(self):
             content_type__app_label='config',
         )
 
+    @classmethod
+    def get_active_metrics(cls):
```

Comment:
A docstring here would be good, eg: returns a list of active metrics (Metric.key).